xml/schema/Bridge/OG.xml

```diff
@@ -24,6 +25,9 @@
     <name>drupal_og_id</name>
     <type>int unsigned</type>
     <comment>Foreign key to the organic group.</comment>
+    <html>
+      <label>Drupal Organic Group</label>
+    </html>
```

Comment:
This file appears to be unused. I think this is from an old module.
Please remove changes to this file. It's not important.